<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="evaluate">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>evaluate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="targetEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>591</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="regexEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>591</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Target">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Regex</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="regexOutcome">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>119</y>
     <width>591</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
